cmd/vclusterctl/cmd/migrate/values.go

```diff
@@ -0,0 +1,112 @@
+package migrate
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/loft-sh/log"
+	"github.com/loft-sh/vcluster/cmd/vclusterctl/flags"
+	"github.com/loft-sh/vcluster/pkg/config/legacyconfig"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+type valuesCmd struct {
+	*flags.GlobalFlags
+	log      log.Logger
+	distro   string
+	filePath string
+	format   string
+}
+
+func migrateValues(globalFlags *flags.GlobalFlags) *cobra.Command {
+	c := &valuesCmd{
+		GlobalFlags: globalFlags,
+		log:         log.GetInstance(),
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:   "values",
+		Short: "Migrates current cluster values",
+		Long: `
+#######################################################
+############### vcluster migrate values ###############
+#######################################################
+Migrates values for a vcluster to the new format
```

Comment:
Not sure if we should be explicit here?
```suggestion
Migrates values for a vcluster to the 0.20 format
```

Reply:
Yes, I think so.